front-end/src/renderer/components/ui/AppRecoveryPhraseWord.vue

```diff
@@ -32,6 +35,29 @@ const handldeWordInput = (e: Event) => {
 
   emit('update:word', value);
 };
+
+const handleKeyDown = (e: KeyboardEvent) => {
+  const inputs = document.querySelectorAll('input[data-testid^="input-recovery-word-"]');
```

Comment:
The AppRecoveryPhraseWord is a component for one input, it should not depend on anything. The handleKeyDown will search for another input, outside the component, which is not desired.